codesniff/codesniff/urls.py

```diff
@@ -1,6 +1,55 @@
 from django.conf.urls import patterns, include, url
 from django.contrib import admin
+from django.contrib.auth.models import User
+from app.models import Code, Score, CodeSmell
+from rest_framework import routers, serializers, viewsets
 
+# Serializers define the API representation.
+class UserSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = User
+        fields = ('username','email')
+
+class CodeSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Code
+        fields = ('title','content','language','creator','date_added')
+
+class CodeSmellSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = CodeSmell
+        fields = ('code','user','line','smell')
+
+class ScoreSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Score
+        fields = ('code','user','score')
+
+# ViewSets define the view behavior.
+class UserViewSet(viewsets.ModelViewSet):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+class CodeViewSet(viewsets.ModelViewSet):
+    queryset = Code.objects.all()
+    serializer_class = CodeSerializer
+
+class CodeSmellViewSet(viewsets.ModelViewSet):
+    queryset = CodeSmell.objects.all()
+    serializer_class = CodeSmellSerializer 
+
+class ScoreViewSet(viewsets.ModelViewSet):
+    queryset = Score.objects.all()
+    serializer_class = ScoreSerializer  
+
+# Routers provide an easy way of automatically determining the URL conf.
+router = routers.DefaultRouter()
+router.register(r'users', UserViewSet)
```

Comment:
Can you make it singular to be consistent? Not really an issue, just a stylistic concern. :)